wu_pipeline/6_visualize/src/build_svg_map.R

```diff
@@ -30,7 +30,7 @@ add_wu_points <- function(svg, wu_dat, svg_huc_locations, season, wu_type_cd, hu
   unique_widths <- sort(unique(wu_dat_svg$stroke_width), decreasing = TRUE)
   for(w in unique_widths) {
     wu_dat_w <- filter(wu_dat_svg, stroke_width == w) 
-    add_wu_data(wu_dot_grp, wu_dat_w, w)
+    add_wu_data(wu_dot_grp, wu_dat_w, w, class = sprintf("wu-dots-%s", wu_type_cd))
```

Comment:
oooh I was close. I wasn't sure how to concat in R, but knew those two variables needed to be put together.

Reply:
lots of ways to do it - `sprintf`, `paste`, `paste0` - but not `cat` funnily. That function just prints to the console 😄